src/registry_schemas/schemas/special_resolution.json

```diff
@@ -19,15 +19,28 @@
                     "format": "date"
                 },
                 "resolution": {
-                    "type": "string"
+                    "type": "string",
+                    "maxLength": 1000
+                },
+                "resolutionDate": {
+                    "type": "string",
+                    "format": "date"
+                },
+                "signingDate": {
+                    "type": "string",
+                    "format": "date"
+                },
```

Comment:
@thorwolpert I've added these two date fields(`resolutionDate` and `signingDate`) but I wasn't sure if `meetingDate` is actually one of these date fields.  If it is, let me know and I'll remove one of the two new date fields and just use `meetingDate`